packages/paste-website/src/components/tokens-list/category-section/index.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import {useVirtual} from 'react-virtual';
+
+import isEqual from 'lodash/isEqual';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+import {Box} from '@twilio-paste/box';
+
+import {AnchoredHeading} from '../../Heading';
+import {TokenCard} from '../token-card';
+import {isMatch, sentenceCase} from '../utils';
+
+import type {TokenCategoryKeys} from '../types';
+
+import {TOKENS_BY_THEME} from '../constants';
+
+const useVirtualizedList = (
+  // @TODO no any
+  items: any[],
+  parentRef: React.MutableRefObject<HTMLDivElement | HTMLElement | null>
+): Pick<ReturnType<typeof useVirtual>, 'scrollToIndex' | 'virtualItems' | 'totalSize'> => {
+  const {scrollToIndex, virtualItems, totalSize} = useVirtual({
+    size: items.length,
+    parentRef,
+    estimateSize: React.useCallback(() => 36, []),
+    overscan: 4,
+    paddingStart: 10,
+  });
+
+  return {scrollToIndex, virtualItems, totalSize};
+};
+
+const Component: React.FC<{
```

Comment:
AI: Please do not use `Component` as a component name, rename to something like `BaseCategorySection`